README.md

```diff
@@ -27,42 +27,103 @@ Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
```

Comment:
for the deployment link, we can replace our master Netlify deployment link instead of the facebook reference.

Reply:
done